.evergreen/run-orchestration.sh

```diff
@@ -37,6 +37,10 @@ if [ -z "$ORCHESTRATION_FILE" ]; then
     ORCHESTRATION_FILE="auth"
   fi
 
+  if [ "$AWS" = "aws" ]; then
+    ORCHESTRATION_FILE="${ORCHESTRATION_FILE}-aws"
+  fi
```

Comment:
I think this is redundant now that we have `ORCHESTRATION_FILE`. Instead of `AWS`:
```
          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} AWS=${AWS} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
```

We can do this:
```
          ORCHESTRATION_FILE=auth-aws.json
...
          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} ORCHESTRATION_FILE=${ORCHESTRATION_FILE} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
```

Reply:
Removed.